core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -361,8 +361,9 @@ var TopicsAndSkillsDashboardPage = function() {
     var editor = element(by.css('.protractor-test-concept-card-text'));
     await waitFor.visibilityOf(
       editor, 'Explanation Editor takes too long to appear');
-
-    await (await browser.switchTo().activeElement()).sendKeys(reviewMaterial);
+    var skillReviewMaterialInput = editor.element(by.css('.oppia-rte'));
+    await skillReviewMaterialInput.click();
+    await skillReviewMaterialInput.sendKeys(reviewMaterial);
```

Comment:
Could you check that `skillReviewMaterialInput` has loaded before interacting with it? The actions.js file makes this easy.

Reply:
Yup Fixed.